client/src/pages/FiltersPage/FiltersPage.js

```diff
@@ -30,23 +29,32 @@ export default function FiltersPage() {
   //tracker for the starred coops, an array of starred coops
   const [starredCoops, setStarredCoops] = React.useState([]);
   const [coopShown, setCoopShown] = React.useState([]);
+  const [dropDownOptions, setDropDownOptions] = React.useState([]);
 
-  async function fetchData() {
+  async function fetchAllCoops() {
     const res = await axios.get('/api/coops');
     setCoops(res.data);
-    setCoopShown(res.data[0]);
+  }
 
+  async function fetchInitialData() {
+    const res = await axios.get('/api/coops');
+    setCoops(res.data);
+    setCoopShown(res.data[0]);
```

Comment:
should've caught this earlier my b but be sure to handle the case where `res.data` is length `0` so this doesn't cause the the site to crash